Automattic/wp-calypso
# Checkout: Fix pre-sales availability

Happy Chat has been disabled in #77024. The unexpected side-effect of that is that we're no longer loading the pre-sales availability from its endpoint. Thankfully, we have a better one now.

## Proposed Changes

* Use the `useMessagingAvailability` hook to get the presales availability.

## Testing Instructions

With a use that does not have any paid plans, try purchasing a paid plan. In the checkout, you should see the Zendesk Messaging launcher (as long as the pre-sales is staffed - check the `is-available` endpoint's return value - hardcode it to true if needed):

<img width="598" alt="Screenshot 2023-05-25 at 18 07 11" src="https://github.com/Automattic/wp-calypso/assets/3392497/0c1fb697-3649-4fea-947e-554482592009">

If there's no availability, make sure the widget is not loaded and shown.
